torch/linalg/__init__.py

```diff
@@ -13,3 +17,248 @@
 
 Alias of :func:`torch.ger`.
 """)
+
+@overload  # noqa: 749
+def norm(input, ord=None, dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
+    # type: (Tensor, str, Optional[List[int]], bool, Optional[Tensor], Optional[int]) -> Tensor
+    pass
+
+@overload  # noqa: 749
+def norm(input, ord=None, dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
+    # type: (Tensor, Optional[number], Optional[List[int]], bool, Optional[Tensor], Optional[int]) -> Tensor
+    pass
+
+@overload  # noqa: 749
+def norm(input, ord=None, dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
+    # type: (Tensor, Optional[number], Optional[int], bool, Optional[Tensor], Optional[int]) -> Tensor
+    pass
+
+@overload  # noqa: 749
+def norm(input, ord=None, dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
+    # type: (Tensor, str, Optional[int], bool, Optional[Tensor], Optional[int]) -> Tensor
+    pass
+
+def norm(input, ord=None, dim=None, keepdim=False, out=None, dtype=None):  # noqa: 749
```

Comment:
`...*, out=None, dtype=None`? Do we ever consistently order out vs dtype? That is, do we always put `out` before dtype or after, or are our functions inconsistent?

Reply:
You're right, `out` and `dtype` should not be positional args, I'll change that.